### PR TITLE
[IMPROVEMENT] Modify the way to handle bad requests, fixes #719

### LINKED ISSUE #719: Installation script is not generating Github_client_id and Github_client_secret in config.py file
Please prefix your issue with one of the following: [BUG], [PROPOSAL], [QUESTION].

Sample platform commit (found at the bottom of each page) : **X.X.X**

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [x] I have checked that the issue I'm posting isn't already reported.
- [x] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/CCExtractor/sample-platform/issues?q=is%3Aissue+is%3Aclosed) and in [opened issues](https://github.com/CCExtractor/sample-platform/issues)
- [x] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

**My familiarity with the project is as follows (check one, eg [X]):**

- [ ] I have never visited/used the platform.
- [ ] I have used the platform just a couple of times.
- [x] I have used the platform extensively, but have not contributed previously.
- [ ] I am an active contributor to the platform.

---

Installation script is not producing Github_client_id and Github_client_secret fields inside config.py file (I had to add those fields manually). Beacuse of that Link to github option is not working as shown in [video](https://drive.google.com/file/d/1WM7sxd3fuIh-Ji2ki4fV3xpwa6givSk2/view?usp=sharing) and also unable to create custom tests. Need to update installtion script, so that it generates both of those fields.

### PR DESCRIPTION
**Pull request by Tarun-Arora**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [ ] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [x] I am an active contributor to the project.

---

There was no 400 error page for the server, and it used the default browser page.
However, at several endpoints, it is necessary to throw a 400 error; for example, `/account/github_callback` should give a 400 error, not a 404.
Screenshot from 2022-10-11 07-16-31
Hence this PR involves the creation of a 400-error page for the same.
Screenshot from 2022-10-11 07-49-43

This PR also includes the modification required in install.sh file suggested by #719 to include the missing parameters in the config file.

### CODE REVIEW

File: config_sample.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 GITHUB_DEPLOY_KEY = ''
 GITHUB_CI_KEY = ''
 GITHUB_CLIENT_ID = ''
-GITHUB_CLIENT_SECRET = ''
+GITHUB_CLIENT_KEY = ''
```
Comment (rdas6313) [inline config_sample.py:19]: Why did you change the name from **GITHUB_CLIENT_SECRET** to **GITHUB_CLIENT_KEY** ?

Comment (Tarun-Arora) [reply config_sample.py:19]: `dodgy` doesn't allow having words such as "secret", it would give errors during GH actions runs.

Comment (rdas6313) [reply config_sample.py:19]: Ok.Got it.

**Review by thealphadollar — APPROVED** [review]
